editor/import/3d/resource_importer_scene.cpp

```diff
@@ -2199,6 +2211,16 @@ void ResourceImporterScene::_add_shapes(Node *p_node, const Vector<Ref<Shape3D>>
 	}
 }
 
+void ResourceImporterScene::_copy_meta(Node *p_src_node, Node *p_dst_node) {
+	List<StringName> meta_list;
+	p_src_node->get_meta_list(&meta_list);
+	for (List<StringName>::Element *E = meta_list.front(); E; E = E->next()) {
+		StringName meta_key = E->get();
```

Comment:
```suggestion
	for (const StringName &meta_key : meta_list) {
```
More convenient approach thanks to iterators

Reply:
ah, nice, thanks!